source/clear-linux/get-started/bare-metal-install-server/bare-metal-install-server.rst

```diff
@@ -241,107 +241,115 @@ on it, and accept the `Default partition schema`_.
 Advanced Configuration
 ----------------------
 
-Use this method to manually configure partitions. These must meet
-`Default partition schema`_. You may also choose `Disk encryption`_ during
-configuration of each partition.
+Use this method to manually configure partitions. Our example follows the
+`Default partition schema`_. Use these defaults, except note that the size
+of your root and any additional partitions may vary.
 
-.. note::
+Select :guilabel:`Advanced Configuration` to launch `cgdisk` and manually
+partition your target media.
+
+Use these partition codes:
+
+* ef00 - EFI System
+* 8200 - Linux swap
+* 8300 - Linux filesystem
+* 8203 - Linux /home
```

Comment:
The last line `8203 - Linux /home` didn't show up for me when I built the html.

Reply:
I removed it. You may have viewed an older commit hash by mistake.

Reply:
cool